app/volunteer/page.tsx

```
@@ -0,0 +1,167 @@
+            <img
+                src="/journey-home-logo.png"
+                alt="Journey Home"
+                className="hidden md:block h-[6em] w-[22em]"
+            />
+
+            <div className="w-full mt-6 md:mt-12 px-6.25 md:px-13.75">
+                <h1 className="text-xl font-medium md:text-2xl md:font-bold text-primary font-raleway">
+                    Volunteering for JourneyHome
+                </h1>
+                <p className="text-sm text-text-1 mt-3">
+                    Welcome to Journey Home's Sign Up System for our weekly Volunteer Pickups, Deliveries, and Organizing Days! We are ending homelessness, please join us and sign up to help! 
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)